ios/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge.m

```diff
@@ -145,4 +145,18 @@ - (instancetype)init
     [GAI sharedInstance].trackUncaughtExceptions = enabled;
 }
 
+RCT_EXPORT_METHOD(setAnonymizeIp:(BOOL)enabled)
+{
+  if(enabled){
```

Comment:
This should also use the `enabled` param rather than just setting `@"1"`. Consider changing the method body to something like:
```
 id<GAITracker> tracker = [[GAI sharedInstance] defaultTracker];
 [tracker set:kGAIAnonymizeIp value:enabled ? @"1" : @"0"];
```